docs/user-guide/usage/node-api.md

```diff
@@ -80,7 +80,7 @@ As `config` contains no relative paths for `extends` or `plugins`, you do not ha
 ```js
 stylelint
   .lint({
-    config: { rules: "color-no-invalid-hex" },
+    config: { rules: { "color-no-invalid-hex": true } },
```

Comment:
Is there a reason why you made this change? Doesn't seem relevant to the rule - was it an autoformat fix or something?

Reply:
Hi @malsf21 and thanks for review. I've encountered broken config in node api user guide (Node v12.19.0 is on my machine if it means).